apps/notification/tests/test_apis.py

```diff
@@ -106,6 +109,36 @@ def create_join_request(self, project, user=None):
         )
         return join_request
 
+    # TODO: To test notification after creation of entry comments
+    def create_project(self):
+        analysis_framework = self.create(AnalysisFramework)
+        return self.create(
+            Project, analysis_framework=analysis_framework,
+            role=self.admin_role
+        )
+
+    # TODO: To test notification after creation of entry comments
+    def create_entry(self, project, **fields):
+        lead = self.create(Lead, project=project)
+        return self.create(
+            Entry, lead=lead, project=lead.project,
+            analysis_framework=lead.project.analysis_framework,
+            **fields
+        )
+
+    # TODO: To test notification after creation of entry comments
+    def create_entry_comment(self, project, user=None):
+        """Create entry comment"""
+        user = user or self.create(User)
+        entry = self.create_entry(project)
+        entry_comment = self.create(
+            EntryComment,
+            entry=entry,
+            assignees=[self.user],
+            text='test comment',
+        )
+        return entry_comment
```

Comment:
Defined but not used.
For notification creation, you will need to use **captureOnCommitCallbacks**. Something like this
https://github.com/the-deep/server/blob/3bebe249667406624a6316e77de3bf4dadfc2832/apps/entry/tests/test_entry_comment.py#L205-L206

Reply:
I've removed this for now